readers/hdfs/src/test/java/com/criteo/hadoop/garmadon/hdfs/writer/ProtoParquetWriterWithOffsetTest.java

```diff
@@ -146,7 +113,7 @@ public void closeRenameFails() throws IOException {
         verify(writerMock, times(1)).close();
 
         reset(writerMock);
-        Assert.assertTrue(thrown);
+        assertTrue(thrown);
         try {
             parquetWriter.close();
         } catch (IOException ignored) {
```

Comment:
The way the test is written, it will throw an IOException because it will try to rename and return false again. This part of the test did not change with my commit. Anyway, the test aims at verifying there is no more interaction with the underlying writer. 
I can enrich the test to verify that once close is called again, the file renaming will be made. Is that the idea under your remark ?

Reply:
I felt like we might want to check that it throws, but maybe not. anyway, FAR